mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java

```diff
@@ -247,11 +247,20 @@ public void onChargeCreatedFailure(String errorMessage) {
         Toaster.show(rootView, errorMessage);
     }
 
+    private int recyclerCount() {
```

Comment:
i don't think it's necessary to create another function just to get the item count of ```mChargesNameListAdapter```.